server/src/routes/fetchTranscript.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { CALLCONNECTION_ID_TO_CORRELATION_ID, TRANSCRIPTION_STORE } from '../utils/callAutomationUtils';
+import { TranscriptionData } from '@azure/communication-call-automation';
+
+const router = express.Router();
+interface FetchTranscriptRequest {
+  serverCallId: string;
+}
+interface FetchTranscriptResponse {
+  transcript: TranscriptionData[];
+}
+
+router.post('/', async function (req, res, next) {
+  const { serverCallId }: FetchTranscriptRequest = req.body;
+  console.log('Fetching transcript for call:', serverCallId, 'available calls:', Object.keys(TRANSCRIPTION_STORE));
+  /**
+   * callId here is the correlationId in the Automation event saying transcription has started
+   * we need to use this to get the call connectionId from the callAutomation client
+   */
+  const connectionId = Object.keys(CALLCONNECTION_ID_TO_CORRELATION_ID).find((key) =>
+    CALLCONNECTION_ID_TO_CORRELATION_ID[key].serverCallId.includes(serverCallId)
```

Comment:
should this be `includes` or `===`?

Reply:
These comparisons should be includes as the callAutomation return for the serverCallId is slightly different than the one from the CallingSDK. They add random amounts of '=' characters to the end of the id in their events so we need to see if the serverCallid from the client is a substring. There is a bug right now from working with the automation engineers where the correlation id's coming from the transcription service are not always right depending on who started the call and started transcription. 

tldr: going with the serverCallId for now but would like to be using the correlationId in the future.

Reply:
> These comparisons should be includes as the callAutomation return for the serverCallId is slightly different than the one from the CallingSDK.

Are the teams aware of this to fix it?